src/functionalTest/java/submitBulk/MCOLJudgement.java

```diff
@@ -0,0 +1,151 @@
+        response.then().statusCode(200);
+        response.prettyPrint();
+    }
+
+    public String removeNamespaces(String xmlResponse) {
+        return xmlResponse.replaceAll("xmlns.*?(\"|\').*?(\"|\')", "")
+                          .replaceAll("(<)(\\w+:)(.*?>)", "$1$3")
+                          .replaceAll("(</)(\\w+:)(.*?>)", "$1$3");
+    }
+
+    @Before
+    public void setUp() {
+        this.xml = soapRequestBody();
+        this.response = soapResponse(xml);
+    }
+
+    public String getCleanResponse() {
+        String xmlResponse = response.asString();
+        xmlResponse = removeNamespaces(xmlResponse);
+        return xmlResponse;
+    }
+
+    @Test
+    public void issueWarrantResponseCode() {
+
+        XmlPath xmlPath = new XmlPath(getCleanResponse()).setRoot("Envelope.Body.bulkResponse");
+        String xmlResponseCode = xmlPath.getString("status.@code");
+        String customerReference = xmlPath.getString("customerReference");
+        String sdtBulkReference = xmlPath.getString("sdtBulkReference");
+        String submittedDate = xmlPath.getString("submittedDate");
+        String sdtService = xmlPath.getString("sdtService");
+        String requestCount = xmlPath.getString("requestCount");
+        int statusCode = response.getStatusCode();
+
+        System.out.println("xmlResponseCode: " + xmlResponseCode);
+        System.out.println("Customer Reference: " + customerReference);
+        System.out.println("SDT Bulk Reference: " + sdtBulkReference);
+        System.out.println("Submitted Date: " + submittedDate);
+        System.out.println("SDT Service: " + sdtService);
+        System.out.println("Request Count: " + requestCount);
+
+        assertThat(xmlResponseCode, equalTo("Ok"));
+        Assertions.assertEquals(200, statusCode);
+
+    }
+
+    @Test
+    public void validateCustRefIssueWarrant() {
+
+        XmlPath xmlPath = new XmlPath(getCleanResponse()).setRoot("Envelope.Body.bulkResponse");
+        String customerReference = xmlPath.getString("customerReference");
+
+        assertThat(customerReference, equalTo(Integer.toString(randomNumber)));
+
+    }
+
+    @Test
+    public void validateSDTBulkRefIssueWarrant() {
+ 
+        XmlPath xmlPath = new XmlPath(getCleanResponse()).setRoot("Envelope.Body.bulkResponse");
+        String sdtBulkReference = xmlPath.getString("sdtBulkReference");
+    
+        assertThat(sdtBulkReference, matchesRegex("MCOL-\\d+-\\d+"));
+    }
+
+    // @Test
+    // public void issueWarrantResponseDescription() {
+
+    //     checkStatusCode(response);
+
+    //     String xmlResponse = response.asString();
+    //     xmlResponse = removeNamespaces(xmlResponse);
+
+    //     XmlPath xmlPath = new XmlPath(xmlResponse).setRoot("Envelope.Body.bulkResponse.status.error");
+    //     String errorDescription = xmlPath.getString("description");
+
+    //     assertThat(errorDescription, containsString("Duplicate User File Reference 91 supplied"));
+    // }
```

Comment:
Is this needed? If the test is valid but can't yet be run because of some other issue then uncomment it and annotate it with the JUnit `@Disabled` annotation. It's also recommended to add a comment to the annotation explaining why it's disabled.

If the test is not needed then delete it.